bcda/models/models.go

```diff
@@ -168,6 +169,30 @@ func (job *Job) GetEnqueJobs(resourceTypes []string, since string, newBeneficiar
 	return enqueJobs, nil
 }
 
+func setJobPriority(acoID string, resourceType string, sinceParam bool) int {
+	var priority int
+	if isSyntheticACO(acoID) {
+		priority = 10 // priority level for jobs for sythetic ACOs that are used for smoke testing
+	} else if resourceType == "Patient" || resourceType == "Coverage" {
+		priority = 20 // priority level for jobs that only request smaller resources
+	} else if sinceParam {
+		priority = 30 // priority level for jobs that only request data for a limited timeframe
+	} else {
+		priority = 100 // default priority level for jobs
+	}
+	return priority
+}
+
+func isSyntheticACO(acoID string) bool {
+	testACOs := []string{"A9990", "A9991", "A9992", "A9993", "A9994"}
```

Comment:
Hard-coding these might lead to maintenance issues.  Should we make this configurable via env var?  
We have something very similar (env var representing a list of ACOs) [here](https://github.com/CMSgov/bcda-app/blob/master/bcda/cclf/cclf.go#L508) (`CCLF_PRIORITY_ACO_CMS_IDS` env var which is defined [here](https://github.com/CMSgov/bcda-ops/blob/master/config/nfs/dev/CCLF_PRIORITY_ACO_CMS_IDS)).  
Would need to create a separate env var for `dev`, `test`, `opensbx`, and `prod`.

Or is there another option?

Reply:
no, that makes the most sense, having them be defined in environment variables